utils/b2i/bufr2ioda_insitu_profile_pirata.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import sys
+from b2iconverter.util import parse_arguments
+from b2iconverter.bufr2ioda_config import Bufr2iodaConfig
+from b2iconverter.bufr2ioda_converter import Bufr2ioda_Converter
+from mbuoyb_tropical_ioda_variables import MbuoybTropicalIODAVariables
+from wmo_codes import *
+
+import numpy as np
+
+
+platform_description = 'PIRATA Tropical mooring profiles from mbuoyb: temperature and salinity'
+
+
+class PirataConfig(Bufr2iodaConfig):
+    def ioda_filename(self):
+        return f"{self.cycle_type}.t{self.hh}z.insitu_profile_pirata.{self.cycle_datetime}.nc4"
+
```

Comment:
Let's switch the suffix to `.nc` in this repo.

Reply:
NCO is good with all netcdf files having extension .nc